docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -42,6 +42,14 @@ Dashboard filters are applied to all the widgets in your dashboard. They consist
 
 When adding a widget or editing an existing one, you will be taken to a new page where you can see the dashboard widget builder. The widget builder has several configuration options that you can use to shape data and add information to your dashboard. Learn more in the [Widget Builder documentation](/product/dashboards/widget-builder/).
 
+### Metrics Extraction
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+If you're using custom tags or custom filters in your widgets using the transaction dataset, we will start collecting metrics to give you accurate counts and rates. During widget creation or editing, you will see approximate data based on indexed event. As soon as you save, we will start extracting metrics for those widgets. It may take up to 15 minutes to start seeing metrics.
```

Comment:
```suggestion
If you're using custom tags or custom filters in your widgets, we will start collecting metrics to give you accurate counts and rates. During widget creation or editing, you will see approximate metrics for transaction datasets based on indexed events. As soon as you save, we will start extracting metrics for those widgets. It may take up to 15 minutes to start seeing metrics.
```